runtime/runtime_controller.h

```diff
@@ -44,6 +44,8 @@ class RuntimeController final : public WindowClient {
 
   bool SetViewportMetrics(const ViewportMetrics& metrics);
 
+  // Stashes the locale_data in window_data_ and then passes the
+  // locale_data to Window::UpdateLocales.
```

Comment:
Avoid describing the implementation, but instead document the intent, any preconditions or restrictions on the values passed in etc.